tests/manage/cluster/nodes/test_nodes_restart.py

```diff
@@ -34,4 +35,61 @@ def test_nodes_restart_aws(self, resources, instances, aws_obj, force):
         aws_obj.restart_ec2_instances(instances=instances, wait=True, force=force)
         self.validate_cluster(resources=resources, nodes=list(instances.values()))
 
+    @pytest.mark.parametrize(
+        argnames=["wait_for_not_ready_nodes"],
+        argvalues=[
+            pytest.param(*[True], marks=pytest.mark.polarion_id("OCS-1138")),
+            pytest.param(*[False], marks=pytest.mark.polarion_id("OCS-1139"))
+        ]
+    )
+    def test_pv_provisioning_under_degraded_state(
+        self, resources, instances, aws_obj, wait_for_not_ready_nodes
+    ):
+        """
+        Test PV provisioning under degraded state
+
+        OCS-1138:
+        - Stop 1 ec2 instance worker node
+        - Wait for the worker node to reach status 'NotReady'
+        - Validate cluster functionality, without checking cluster and Ceph
+          health
+          by creating resources and running IO
+        - Start the worker node ec2 instance
+        - Check cluster and Ceph health
+
+        OCS-1139:
+        - Stop 1 ec2 instance worker node
+        - Without waiting for the worker node to reach status 'NotReady' -
+        - Validate cluster functionality, without checking cluster and Ceph
+          health
```

Comment:
same .. L64 and &65 can be on merged.. right ?

Reply:
will fix